var/spack/repos/builtin/packages/libpng/package.py

```diff
@@ -57,7 +57,7 @@ def configure_args(self):
             # not honored, see
             #   https://sourceforge.net/p/libpng/bugs/210/#33f1
             # '--with-zlib=' + self.spec['zlib'].prefix,
-            'CFLAGS=-I{0}'.format(self.spec['zlib'].prefix.include),
-            'LDFLAGS=-L{0}'.format(self.spec['zlib'].prefix.lib)
```

Comment:
note: this changes from `-L/path` to `-L/path -lz`

Reply:
I guess the `-lz` is hard-coded in the makefile somewhere.

Reply:
Instead of `ld_flags` you can use `search_flags`. That should expand to just the `-L<path>` part of the linker command.

Reply:
oh, let's use search_flags! :)